Talent show

ClarenceAtim opened the pull request:
The following key featues where added

         Images
        Index.html
        Interfaces
        scripts
        style

ClarenceAtim committed:
i acreated  Images/
        Index.html
        Interfaces/
        scripts/
        style/

ClarenceAtim committed:
modified:   Index.html
        modified:   Interfaces/About.html
        modified:   style/About.css
        modified:   style/homepageStyle.css

ClarenceAtim committed:
modified:   README.md

ClarenceAtim committed:
edit index html file

ClarenceAtim committed:
Rename Index.html to index.html

ClarenceAtim committed:
modified index.html

ClarenceAtim committed:
Merge branch 'talent-show' of https://github.com/ClarenceAtim/Capstone-project into talent-show

ClarenceAtim committed:
modified:   index.html

ClarenceAtim committed:
fix link errors

ClarenceAtim committed:
.eslintrc.json
        .github/
        .gitignore
        .stylelintrc.json

ClarenceAtim committed:
modified css

ClarenceAtim committed:
modified hesder

ClarenceAtim committed:
modified hesder

ClarenceAtim committed:
modified hesder

ClarenceAtim committed:
modified read.me file

ClarenceAtim committed:
modified html

ClarenceAtim committed:
modified read.me

AlphaNtihinduka commented on index.html:
```diff
@@ -0,0 +1,152 @@
+            Talent Show          
+          </h2>
+          
+               <h1 class="captiontext">
+                Presenting!<br> All talent from the street this 2023
+               </h1>
```
- [ ] You have done a very great job to ensure that you are using `h1` HTML tag for the heading of the page. However, please try as much as possible to ensure that there is an image in the title text to ensure that your design matches the original design and also reduce the height of the heading. In addition, this will improve the UI of your application 👍
- [ ] In addition, consider summarizing the heading of the page up to 3 lines. This will improve the SEO of your application.

Your design 😎 
![image](https://github.com/ClarenceAtim/Capstone-project/assets/99394374/ab77f86f-f7bb-4983-a185-c34f3c05b8f6)

Original design 😎 
![image](https://github.com/ClarenceAtim/Capstone-project/assets/99394374/41bbce13-6b05-4641-8f0f-b92cdbffa512)

AlphaNtihinduka commented on style/main.css:
```diff
@@ -0,0 +1,254 @@
+    background-color: rgb(251, 243, 243);
+  }
+
+  .brand-logo {
+    width: 8rem;
+    display: block;
+  }
```
- [ ] I have noticed that your brand logo height is large and therefore makes the navigation bar bigger. Kindly consider reducing the height of the logo to match the original design ☺️. This will also make your project have improved UI 👍.

Your design 😎 
![image](https://github.com/ClarenceAtim/Capstone-project/assets/99394374/c817a861-57aa-40f5-9551-843f8dd05087)

Original design 😎 
![image](https://github.com/ClarenceAtim/Capstone-project/assets/99394374/c183ecba-467a-450c-9351-c1a320fedd01)

AlphaNtihinduka commented on index.html:
```diff
@@ -0,0 +1,152 @@
+          <article class="shedule-cards">
+            <img class="image" src="Images/dance 1.png"  alt="dance-icon ">
+            <h3>Openning with dance</h3>
+            <p class="text1">watch amazing dance talents from douala. Give you what they do best in their career</p>
```
- [ ] Please make sure that your paragraph matches the original design provided. If you notice the color used is close to gray. Therefore I suggest that you consider updating the color of the paragraph of all the cards in the shedule-container ☺️.

Your design 😎 
![image](https://github.com/ClarenceAtim/Capstone-project/assets/99394374/b716a40f-64bf-4f2d-8147-c2d38c9ac438)

Original design 😎 
![image](https://github.com/ClarenceAtim/Capstone-project/assets/99394374/84b458c7-aa09-49ae-b4bf-d772f942d1bf)

AlphaNtihinduka commented on scripts/homepage.js:
```diff
@@ -0,0 +1,57 @@
+    </div>
+    <div class="cards">
+     <h2>${cardArtist.name}</h2>
+        <h3>${cardArtist.title}</h3>
+
+        <p class="demo-text">${cardArtist.description}</p>
```
- [ ] If you notice carefully, the artist description is italic. Kindly apply italic property on them to match the original design ☺️.

You design 😎 
![image](https://github.com/ClarenceAtim/Capstone-project/assets/99394374/5fb32e64-7eb2-44b0-8233-61600ac0639d)

Original design 😎 
![image](https://github.com/ClarenceAtim/Capstone-project/assets/99394374/4d16387e-0933-411e-aa4f-f2fa2fead4ef)

AlphaNtihinduka commented on scripts/homepage.js:
```diff
@@ -0,0 +1,57 @@
+    title: 'Main Dj',
+    description: 'Retrouvez les meilleurs HITS Camerounais de 2018 à 2020 dans un Mashup comme vous les aimez.',
+    Image: 'Images/peter.png',
+  },
```
- [ ] To make the container professional and also make your design match the original design, I suggest to consider adding more 1 artist to have 6 artists. This will also improve the UI of your application ☺️.

Your design 😎 
![image](https://github.com/ClarenceAtim/Capstone-project/assets/99394374/6a622505-dfc3-470f-8447-1e70bf8d093b)

AlphaNtihinduka commented on Interfaces/About.html:
```diff
@@ -0,0 +1,133 @@
+          </div>
+        </section>
+  
+        <footer>
+          <div>
+            <a href="../index.html"><img class="footer-image" src="../Images/logo.png"></a>
+          </div>
+          <div>
+            <p>2023 Talent show copyright &copy;</p>
+            <p>Some right reserved</p>
+          </div>
+        </footer>
```
- [ ] If you notice from the original design, the footer of the about page is dark in desktop version and white in mobile version. Kindly try as much as possible to apply the changes so that our design sticks to the original design provided 👍.

Your design 😎 
![image](https://github.com/ClarenceAtim/Capstone-project/assets/99394374/42ab70cc-e8ab-4324-8c47-ec440a82cba3)

Original design 😎 
![image](https://github.com/ClarenceAtim/Capstone-project/assets/99394374/55183ed5-df78-43fb-b659-b9ff5fd1e8d9)

AlphaNtihinduka reviewed (changes requested):
## Hi @ClarenceAtim 🙋‍♂️
Good job working on your Talent show application. You have done the design to the best expectation and incredibly well. 👏  

You still need to implement few on the project but you are almost there!!! 🚶 : 

TO HIGHLIGHT:
- Proper use of HTML semantic tags ✔️
- Displayed the speakers dynamically ✔️
- Mobile menu working properly ✔️
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

![close](https://user-images.githubusercontent.com/99394374/210771148-619a4265-33b2-4e68-8901-6bb52693d6de.gif)

# Required Changes :recycle: 
- _Kindly consider the inline review comments._

## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

ClarenceAtim committed:
new file:   Images/headerbg.png.jpg
        new file:   Images/main mc.png
        modified:   scripts/homepage.js
        modified:   style/About.css
        modified:   style/homepageStyle.css
        modified:   style/main.css

OLIPLICHE commented on scripts/homepage.js:
```diff
@@ -0,0 +1,63 @@
+const seeArtist = document.querySelector('.artist-section');
+const artsitSection = document.querySelector('.artist-container');
+
+seeMore.addEventListener(('click'), () => {
+  seeArtist.classList.toggle('setoff');
+});
+
+const artistContainer = [
+  {
+    name: 'ELLA',
+    title: 'Main artist',
+    description: 'Answering the call of nostalgia and in a bid to bridge the gap between the new-school gospledance and socia dance.',
+    Image: 'Images/main artiest .png',
+  },
+  {
+    name: 'mr.kah',
+    title: '3th Artist',
+    description: 'Trap Rapper/Songwriter from Bamenda, Cameroon. Currently signed to B.D.A GANG MUSIC TRAP STAR! VOICE OF THE STREETS! CAMEROON!',
+    Image: 'Images/mr kah.png',
+  },
```
- Hi, you have done an incredible job and you are almost there. As stated by the previous reviewer, I would advise you to do as much as possible to match the original design. I marked red in the original picture to attract your attention to where you need to improve. 
- NB: Use the exact same color, text, font size, backgrounds, border, etc ...
- Your design:
![Capture](https://github.com/ClarenceAtim/Capstone-project/assets/63972670/de106607-e386-4e18-9afd-a8dfc3abe958)

- Original design:

![Capture1](https://github.com/ClarenceAtim/Capstone-project/assets/63972670/6a50c85b-66f5-4501-897d-c4166c390c4d)

OLIPLICHE reviewed (changes requested):
Hi @ClarenceAtim, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Built Home and About pages 👍 
- Have desktop and mobile sizes 👍 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ClarenceAtim commented on scripts/homepage.js:
```diff
@@ -0,0 +1,63 @@
+const seeArtist = document.querySelector('.artist-section');
+const artsitSection = document.querySelector('.artist-container');
+
+seeMore.addEventListener(('click'), () => {
+  seeArtist.classList.toggle('setoff');
+});
+
+const artistContainer = [
+  {
+    name: 'ELLA',
+    title: 'Main artist',
+    description: 'Answering the call of nostalgia and in a bid to bridge the gap between the new-school gospledance and socia dance.',
+    Image: 'Images/main artiest .png',
+  },
+  {
+    name: 'mr.kah',
+    title: '3th Artist',
+    description: 'Trap Rapper/Songwriter from Bamenda, Cameroon. Currently signed to B.D.A GANG MUSIC TRAP STAR! VOICE OF THE STREETS! CAMEROON!',
+    Image: 'Images/mr kah.png',
+  },
```
the previous reviewer talk about intalic

ClarenceAtim replied:
i did everything that was require by the previouse reviewer so i dont understand what you are talking about

ClarenceAtim committed:
modified scripts and css

Ismailco reviewed (approved):
### Hi @ClarenceAtim, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

![cr-approve](https://user-images.githubusercontent.com/25227022/165483662-7b06c812-4cc0-40b9-a84c-4729d4cf10c3.gif)

## Highlights

❌ Kindly don't remove reviewer comments
✔️ No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

Good luck! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

bimbolabuari reviewed (commented):
Hello @ClarenceAtim  👋,

_This is an additional review due to the second opinion request._

It appears that your project has been approved, and the issue has been resolved. If you require further clarification on any of the reviews, please reply to this message and tag me.

Keep rocking 🚀!
